### PR TITLE
Update using-packages.md

### PR DESCRIPTION
**Pull request by mit-mit**
Addressing https://github.com/dart-lang/site-www/issues/421#issuecomment-342307259

### CODE REVIEW

File: using-packages.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ platform SDKs (like
 
 If you are looking to develop a new package, please see [developing packages](/developing-packages/).
 
+If you are looking to add assets, images, or fonts to your application, please
```
Comment (sethladd) [inline using-packages.md]: maybe hint as to why this is related to Using Packages ?

e.g. "Some package provide assets such as fonts and images. Learn more...."

**Comment by sethladd** [comment]
lgtm but I think we can clarify the relationship between packages and assets, especially for the "I'm using a package that has an asset, and I want to load and use that asset" use case.

File: using-packages.md (hunk 1 above)
Comment (mit-mit) [reply using-packages.md]: Would just "If you are looking to add assets, images, or fonts, whether stored in files or packages, please..." work?

Comment (sethladd) [reply using-packages.md]: sure!

**Comment by mit-mit** [comment]
thanks for the quick review!

**Pull request merged**